doc/release/1.17.0-notes.rst

```diff
@@ -214,16 +226,21 @@ This change is backwards compatible, but now allows code like::
 New Features
 ============
 
-New extensible random module with selectable random number generators
----------------------------------------------------------------------
-A new extensible random module along with four selectable random number
+.. currentmodule:: numpy.random
+
+New extensible `numpy.random` module with selectable random number generators
+-----------------------------------------------------------------------------
+A new extensible `numpy.random` module along with four selectable random number
 generators and improved seeding designed for use in parallel processes has been
-added. The currently available bit generators are MT19937, PCG64, Philox, and
-SFC64. PCG64 is the new default while MT19937 is retained for backwards
+added. The currently available :ref:`Bit Generators <bit_generator>` are
+`~mt19937.MT19937`, `~pcg64.PCG64`, `~philox.Philox`, and `~sfc64.SFC64`.
+``PCG64`` is the new default while ``MT19937`` is retained for backwards
 compatibility. Note that the legacy random module is unchanged and is now
-frozen, your current results will not change. Extensive documentation for the
-new module is available online at
-`NumPy devdocs <http://www.numpy.org/devdocs/reference/random/index.html>`_.
+frozen, your current results will not change. More information is available in
+the :ref:`API change description <new-or-different>` and in the `top-level view
```

Comment:
Hmm, not sure this is better as `:ref:` doesn't show the link to someone reading the text or viewing on github. In fact, I think we need to decide whether we are aiming at the rendered html doc, which I suspect no one currently reads, or the text. I suppose that once everything gets set up we can point folks to the generated docs when sending out the announcement. When are the online docs generated?

Reply:
It would certainly make the github doc easier if the online docs are available, we could just post a link instead of the release note.

Reply:
Decision needed. This PR assumes the target audience uses the rendered documentation

The [devdocs on numpy.org](http://www.numpy.org/devdocs/) always point to the master branch. Right now the canonical location for the docs is https://docs.scipy.org/doc/ which is updated via https://github.com/scipy/scipy.org together with uploading the documentation.

Reply:
What about devdocs? I can see making all the fixes in the master branch and then backporting them just before a release, although the extra step invites process errors.

I can see that at some point we will need  "HOWTO_WRITE_RELEASE_NOTES" documentation. Maybe make this a topic for discussion tomorrow.